packages/as-transformer/src/transformer.ts

```diff
@@ -75,85 +73,28 @@ export class Transformer extends TransformVisitor {
         !source.internalPath.startsWith(`node_modules/`) &&
         !utils.isLibrary(source),
     );
-
-    let additionalImports = new Map<string, boolean>();
-
     sources.forEach((source) => {
-      resetUpdates();
+      TransformUpdates.resetUpdates();
       this.visit(source);
-
-      let content = source.text;
-      let neededImports = new Map<string, boolean>();
-
-      const updates = getUpdates();
-
-      if (updates.length > 0) {
-        updates.forEach((update) => {
-          const token = 'export function ';
-          const index = content.indexOf(token, update.begin) + token.length;
-
-          assert(
-            index >= token.length,
-            `exported function not found in file ${source.internalPath}` +
-              `, but decorator ${protobufTransformerDecorator} was.`,
-          );
-          content.replace(
-            token + update.funcToPrivate,
-            'function _' + update.funcToPrivate,
+      let actualSource = source;
+
+      for (let transformer of functionTransformers) {
```

Comment:
Same. If it's done and it's working, you can keep it but it's not mandatory right now. We should focus on getting shit done. We will make it more evolutive later.